x/dex/types/message_place_limit_order_test.go

```diff
@@ -109,7 +116,44 @@ func TestMsgPlaceLimitOrder_ValidateBasic(t *testing.T) {
 			err: dextypes.ErrTickOutsideRange,
 		},
 		{
-			name: "valid msg",
+			name: "price < minPrice",
+			msg: dextypes.MsgPlaceLimitOrder{
+				Creator:        sample.AccAddress(),
+				Receiver:       sample.AccAddress(),
+				TokenIn:        "TokenA",
+				TokenOut:       "TokenB",
+				LimitSellPrice: &TINYDEC,
+				AmountIn:       math.OneInt(),
+			},
+			err: dextypes.ErrPriceOutsideRange,
+		},
+		{
+			name: "price > maxPrice",
+			msg: dextypes.MsgPlaceLimitOrder{
+				Creator:        sample.AccAddress(),
+				Receiver:       sample.AccAddress(),
+				TokenIn:        "TokenA",
+				TokenOut:       "TokenB",
+				LimitSellPrice: &HUGEDEC,
+				AmountIn:       math.OneInt(),
+			},
+			err: dextypes.ErrPriceOutsideRange,
+		},
+		{
+			name: "price > maxPrice",
```

Comment:
description and error do match
"price > maxPrice" vs err: dextypes.ErrInvalidPriceAndTick